.github/workflows/go.yml

```
@@ -103,7 +103,13 @@ jobs:
               working-directory: go
               run: |
                   make install-tools-go${{ matrix.go }} build
-                  make -k unit-test integ-test example-test
+                  make -k unit-test integ-test
+
+            - name: Run Example Tests
+              working-directory: go
+              if: ${{ matrix.engine.version != '6.2' }}
```

```suggestion
              if: ${{ matrix.engine.version == '8.0' }}
```
We have tests on 7.0, while some commands added in 7.2 or in 8.+